cleo/opto/opsin_library.py

```diff
@@ -74,6 +74,267 @@ def chr2_4s() -> FourStateOpsin:
     )
 
 
+def ReaChR_4s() -> BansalFourStateOpsin:  # include this if it's a 4 state
+    """Returns a 4-state ReaChR model.
+
+    Params given in Bansal et al., 2020. [Table 1]
+    Action spectrum from `Krause et al., 2017, Fig. 1b
+    <https://doi.org/10.1016/j.bpj.2017.02.001>`_,
+    extracted using `Plot Digitizer <https://plotdigitizer.com/>`_.
+    """
+
+    return BansalFourStateOpsin(
+        Gd1=7.7e-3 / ms,
+        Gd2=1.25e-3 / ms,
+        Gr0=3.33e-5 / ms,
+        g0=14.28 * nsiemens,
+        phim=5e17 / mm2 / second,
+        k1=1.2 / ms,
+        k2=0.01 / ms,
+        Gf0=0.0005 / ms,
+        Gb0=0.0005 / ms,
+        kf=0.012 / ms,
+        kb=0.001 / ms,
+        gamma=0.05,
+        p=1,
+        q=1,
+        E=7 * mV,
+        name="ReaChR",
+        spectrum=[
+            (479, 0.67),
```

Comment:
the spectrum needs to be in order. Is there a reason this 479 is here in front? Is it a typo?